docker-selenium.yaml

```diff
@@ -111,39 +117,108 @@ subpackages:
         - supervisor-config
     pipeline:
       - runs: |
-          mkdir -p ${{targets.subpkgdir}}/etc
-          mv ${{targets.destdir}}/etc/supervisord.conf ${{targets.subpkgdir}}/etc
-          mv ${{targets.destdir}}/etc/supervisor ${{targets.subpkgdir}}/etc
+          mkdir -p ${{targets.contextdir}}/etc
+          mv ${{targets.destdir}}/etc/supervisord.conf ${{targets.contextdir}}/etc
+          mv ${{targets.destdir}}/etc/supervisor ${{targets.contextdir}}/etc
 
           # Unset setuid bit
           # jason@ copied this directly from https://serverfault.com/a/238964
           # The extra 0 is to unset the setuid bit apparently.
-          chmod 00775 ${{targets.subpkgdir}}/etc/supervisord.conf
+          chmod 00775 ${{targets.contextdir}}/etc/supervisord.conf
+
+  - name: ${{package.name}}-standalone
+    dependencies:
+      runtime:
+        - selenium-server-standalone
+        - ${{package.name}}
+        - ${{package.name}}-config
+    pipeline:
+      - working-directory: Standalone
+        pipeline:
+          - runs: |
+              mkdir -p ${{targets.contextdir}}/etc/supervisor/conf.d
+              mkdir -p ${{targets.contextdir}}/opt/bin
+              cp start-selenium-standalone.sh ${{targets.contextdir}}/opt/bin/start-selenium-standalone.sh
```

Comment:
Side of side comment: you can omit the target filename `start-selenium-standalone.sh` :-)